Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -42,9 +43,17 @@ private void GetLobby()
             var lobby = task.GetAwaiter().GetResult();
             if (lobby != CurrentLobby)
             {
+                var playerIds = new string[4];
+                var index = 0;
+                foreach (var player in lobby.Players)
+                {
+                    playerIds[index] = player.Id;
+                    index++;
+                }
                 if (lobby.Players.Count > 1)
                 {
                     playButton.gameObject.SetActive(true);
+                    playButton.PlayerIds = playerIds.ToList();
                 }
 
                 foreach (var player in lobby.Players)
```

Comment:
We can combine our two foreach loops together here, rather than looping through the same collection twice.